### PR TITLE
ENG-640: add get content version details API

### PR DESCRIPTION
**Pull request by alepintus**
Hi @BrenoQVDS @eugeniosant @Kerruba @gidesan  ,
added get content version details API and get content version list optimization
Could you please have a look?
Thanks

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/web/contentsversioning/validator/ContentVersioningValidator.java [hunk 1]
```diff
@@ -36,4 +37,16 @@ public boolean contentVersioningExist(String contentId){
...
 
+    public boolean checkContentIdForVersion(String contentId, Long versionId) {
+        boolean validationResult = false;
+        try {
+            final ContentVersion version = versioningManager.getVersion(versionId);
+            if (version.getContentId().equals(contentId)){
+                return true;
+            }
+        } catch (ApsSystemException e) {
+            e.printStackTrace();
```
Comment (eugeniosant) [inline plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/web/contentsversioning/validator/ContentVersioningValidator.java:48]: Can you substitute this console output with a right log?

File: plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/services/contentsversioning/ContentVersioningService.java [hunk 2]
```diff
@@ -36,21 +38,48 @@
...
+    }
+
+    public ContentDto getContent(Long versionId) {
+        try {
+            final ContentVersion contentVersion = versioningManager.getVersion(versionId);
+            final Content content = versioningManager.getContent(contentVersion);
+            final ContentDto contentDto = contentService.getDtoBuilder().convert(content);
+            return contentDto;
+        } catch (ApsSystemException e) {
+            e.printStackTrace();
```
Comment (eugeniosant) [inline plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/services/contentsversioning/ContentVersioningService.java:78]: Can you substitute this console output with a right log?

File: plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/web/contentsversioning/validator/ContentVersioningValidator.java (hunk 1 above)
Comment (alepintus) [reply plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/web/contentsversioning/validator/ContentVersioningValidator.java:48]: Done

File: plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/services/contentsversioning/ContentVersioningService.java (hunk 2 above)
Comment (alepintus) [reply plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/services/contentsversioning/ContentVersioningService.java]: Done

**Comment by entando-service** [comment]
All is Well

**Pull request merged**